testsetups/remote_control.go

```diff
@@ -41,6 +41,7 @@ func StartRemoteControlServer(name string, stopChan chan struct{}) {
 
 	// Start listening for shutdown calls
 	go func() {
+		//nolint:gosec
```

Comment:
if not needed can this be removed?

Reply:
This issue has been open since 2016 and still has no resolution: https://github.com/golang/go/issues/16100
So I skipped it with the linter